### PR TITLE
Upgrade spectrometer Lumi components

### PR DESCRIPTION
**Pull request by dhevang**
### Briefly, what does this PR introduce?

Adds and upgrades components to the Lumi spectrometer system in the far-bwd region.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

File: compact/far_backward/lumi/spec_homo_cal.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   </detectors>
 
   <readouts>
-    <readout name="LumiSpecCALHits">
+    <readout name="EcalLumiSpecHits">
```
Comment (wdconinc) [inline compact/far_backward/lumi/spec_homo_cal.xml:21]: Does it make sense to keep `Lumi` up front here for lexical sorting among the other lumi collections?

File: src/LumiCollimator_geo.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  double         width   = box_dim.attr<double>(_Unicode(x));;
+  double         depth   = box_dim.attr<double>(_Unicode(z));;
+
+  Box box_outer(width, height, depth);
+
+  // Create inner box
+  xml::Component box_dim_2 = x_det.child(_Unicode(dimensions_inner));
+  double         height_2  = box_dim_2.attr<double>(_Unicode(y));;
+  double         width_2   = box_dim_2.attr<double>(_Unicode(x));;
+  double         depth_2   = box_dim_2.attr<double>(_Unicode(z));;
```
Comment (wdconinc) [inline src/LumiCollimator_geo.cpp:39]:
```suggestion
  double         height  = box_dim.attr<double>(_Unicode(y));
  double         width   = box_dim.attr<double>(_Unicode(x));
  double         depth   = box_dim.attr<double>(_Unicode(z));

  Box box_outer(width, height, depth);

  // Create inner box
  xml::Component box_dim_2 = x_det.child(_Unicode(dimensions_inner));
  double         height_2  = box_dim_2.attr<double>(_Unicode(y));
  double         width_2   = box_dim_2.attr<double>(_Unicode(x));
  double         depth_2   = box_dim_2.attr<double>(_Unicode(z));
```

**Review by wdconinc — COMMENTED** [review]
Just a few comments for now. Rest looks good too.

File: compact/far_backward/lumi/spec_homo_cal.xml (hunk 1 above)
Comment (dhevang) [reply compact/far_backward/lumi/spec_homo_cal.xml:21]: That was my first line of thinking but then I saw a different "convention" from other CAL readouts.  They seemed to first start with the type: Ecal or Hcal, and then the name of the detector.

Comment (dhevang) [reply compact/far_backward/lumi/spec_homo_cal.xml:21]: https://github.com/eic/epic/blob/9eb69289d9f781b10f45109937fa2e3fca797b6b/compact/ecal/backward_PbWO4.xml#L81

File: src/LumiCollimator_geo.cpp (hunk 2 above)
Comment (dhevang) [reply src/LumiCollimator_geo.cpp]: Done.

File: compact/far_backward/lumi/spec_homo_cal.xml (hunk 1 above)
Comment (wdconinc) [reply compact/far_backward/lumi/spec_homo_cal.xml:21]: Yeah, but we've sort of avoided that convention for far forward and far backward... There the overall system is more pertinent.

Comment (dhevang) [reply compact/far_backward/lumi/spec_homo_cal.xml]: Ok, changed the readout naming scheme back to the way I had it before.

**Review by wdconinc — APPROVED** [review]

**Comment by dhevang** [comment]
I don't quite understand why the two required benchmark runs failed (detector_benchmarks and physics_benchmarks).  Looking into the log details, I see "ERROR: No files to upload".  Which files are being searched for?

**Pull request merged**